ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |-
+    FROM quay.io/fuse_qe/syndesisqe-tests:1.15.x
+    USER root
+    RUN apt install -y rsync
+    RUN chgrp -R 0 /home/seluser && \
+    chmod -R g=u /home/seluser && \
+    chgrp 0 / && \
+    chmod g=u / && \
+    chmod g=u /etc/passwd /etc/group && \
+    chgrp -R 0 /tmp && \
+    chmod -R g=u /tmp
+    USER seluser
+  to: fuse-online-test-runner
+- dockerfile_literal: |-
+    FROM cli
+    RUN dnf update -y && dnf install httpd-tools -y && dnf install rsync -y && dnf clean all
+  from: cli
+  to: htpasswd-cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 2Gi
+tests:
+- as: fuse-online-interop-aws
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
```

Comment:
We no longer need to use the jira-epic LPTOCPCI-290 now that Aditya has created the firewatch ticket tracking dashboard https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12353648

Reply:
Removed jira-epic from firewatch configurations